lib/ci-stack.ts

```diff
@@ -149,34 +166,15 @@ export class CIStack extends Stack {
     const listenerCertificate = ListenerCertificate.fromArn(certificateArn.secretValue.toString());
     const agentNode = new AgentNodes(this);
 
-    if (useProdAgents.toString() === 'true') {
-      // eslint-disable-next-line no-console
-      console.warn('Please note that if you have decided to use the provided production jenkins agents then '
+    // eslint-disable-next-line no-console
+    console.warn('Please note that if you have decided to use the provided production jenkins agents then '
```

Comment:
Isn't the conditional `useProdAgents` required here?
The warning was only relevant if the ProdAgents parameter is enabled.